contrib/sequoia-git/template.py

```diff
@@ -0,0 +1,31 @@
+pkgname = "sequoia-git"
+pkgver = "0.1.0"
+pkgrel = 0
+build_style = "cargo"
+make_check_env = {"TARGET": self.profile().triplet}
+hostmakedepends = ["cargo", "pkgconf"]
+makedepends = [
+    "bzip2-devel",
+    "libgit2-devel",
+    "nettle-devel",
+    "openssl-devel",
+    "rust-std",
+    "sqlite-devel",
+]
+checkdepends = ["git", "gnupg"]
+pkgdesc = "Tool to verify git commit signatures based on a policy"
+maintainer = "Jan Christian Grünhage <jan.christian@gruenhage.xyz>"
+license = "LGPL-2.0-or-later"
+url = "https://gitlab.com/sequoia-pgp/sequoia-git"
+source = f"https://gitlab.com/sequoia-pgp/sequoia-git/-/archive/v{pkgver}/sequoia-git-v{pkgver}.tar.gz"
```

Comment:
```suggestion
source = f"{url}/-/archive/v{pkgver}/sequoia-git-v{pkgver}.tar.gz"
```

(only for when it starts with url)